build.fsx

```diff
@@ -323,7 +323,7 @@ Target "MultiNodeTests" <| fun _ ->
                 |> append assembly
                 |> append "-Dmultinode.enable-filesink=on"
                 |> append (sprintf "-Dmultinode.output-directory=\"%s\"" testOutput)
-                |> appendIfNotNullOrEmpty spec "-Dmultinode.test-spec="
+                |> appendIfNotNullOrEmpty spec "-Dmultinode.spec="
```

Comment:
Allows the ability for the MNTR to correctly run a single spec from the FAKE commandline, i.e.

```
 ./build.cmd multinodetests spec=ClusterBroadcastGroup
```